cluster/images/etcd-empty-dir-cleanup/Dockerfile

```diff
@@ -12,12 +12,15 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-FROM gliderlabs/alpine
+FROM busybox
```

Comment:
The problem with busybox is we don't currently have a way to track CVEs. It's not so much an issue with this particular image, but something to think about... 

I'm tempted to put the version number in here (`busybox:1.26.2`). On the one hand, it would make it easier to see which version images were on. On the other hand, it makes it easier for the versions to be left behind.... what are your thoughts?